es-base-styles/main/default/lwc/imageGallery/__tests__/imageGallery.test.js

```diff
@@ -0,0 +1,81 @@
+import { createElement } from 'lwc';
+import ImageGallery from 'c/imageGallery';
+
+// Realistic data with a list of spaces
+const mockSpacesList = require('./data/getRelatedSpaces.json');
+
+describe('c-image-gallery', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('displays image tiles', () => {
+        // Create element
+        const element = createElement('c-image-gallery', {
+            is: ImageGallery
+        });
+
+        element.items = mockSpacesList;
+        document.body.appendChild(element);
+
+        return Promise.resolve().then(() => {
+            const lightningLayoutEl = element.shadowRoot.querySelector(
```

Comment:
Drop that layout check, we don't need to test that static part of the layout. We need only to check the image tile.

Reply:
Done